source/develop/release-management/features/network/liveMigrationSupportForSRIOV.html.md

```diff
@@ -82,4 +82,58 @@ the temporary disappearance of the SR-IOV device.
    
 When migration finishes and the SR-IOV device is restored, it should be 
 reattached to that bond. In the context of this feature, we would supply
-the el7 hooking mechanism to make it happen seamlessly.
+the el7 hooking mechanism to make it happen seamlessly in the future, while
+currently migration is possible using the following procedure via NetworkManager:
+
+
+Create the bond (with ens3 being the passthrough interface in this example):
+```
+~]$ nmcli con add type bond con-name bond0 ifname bond0 mode active-backup primary ens3
```

Comment:
~]$ nmcli con add type bond con-name bond0 ifname bond0 mode active-backup primary ens3
=>
    ~]$ nmcli con add type bond con-name bond0 ifname bond0 mode active-backup primary ens3
With spaces in front it will be highlighted as code. For example check how the commands are highlighted here:
https://www.ovirt.org/blog/2016/11/ovirt-provider-ovn/